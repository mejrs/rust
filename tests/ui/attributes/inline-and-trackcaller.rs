//! Regression test for <https://github.com/rust-lang/rust/issues/142783>.

#![crate_type = "lib"]

#[inline] //~ERROR attribute should be applied to function or closure
#[track_caller]
macro_rules! contained {
    () => {};
}
