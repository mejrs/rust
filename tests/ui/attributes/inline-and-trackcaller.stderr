error[E0518]: attribute should be applied to function or closure
  --> $DIR/inline-and-trackcaller.rs:5:1
   |
LL | #[inline]
   | ^^^^^^^^^ not a function or closure

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0518`.
